package.json

```diff
@@ -26,8 +26,16 @@
   "license": "MIT",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "module": "dist/index.mjs",
+  "exports": {
+    "import": "./dist/index.mjs",
+    "require": "./dist/index.js"
+  },
```

Comment:
Never seen this before, got any documentation how this works, what it's for?

Reply:
https://nodejs.org/api/packages.html#packages_exports